ui/components/app/approve-content-card/approve-content-card.js

```diff
@@ -134,6 +134,20 @@ export default function ApproveContentCard({
                   flexDirection={FLEX_DIRECTION.COLUMN}
                   className="approve-content-card-container__transaction-details-extra-content"
                 >
+                  <Box
+                    display={DISPLAY.FLEX}
+                    justifyContent={JustifyContent.spaceBetween}
+                  >
+                    <Text
+                      variant={TextVariant.bodySm}
+                      fontWeight={FontWeight.Normal}
+                      color={TextColor.textMuted}
+                      as="h6"
+                    >
+                      <span>{t('transactionDetailLayer2GasHeading')}</span>
+                      {`${ethTransactionTotal} ${nativeCurrency}`}
+                    </Text>
+                  </Box>
```

Comment:
Should this be here? Was it a bad merge? The task was to swap `Typography` for `Text` here you are adding new UI

<img width="1440" alt="Screenshot 2023-04-28 at 3 56 03 PM" src="https://user-images.githubusercontent.com/8112138/235071179-7b966af7-5d72-4dd5-9b31-10bdbbdcde05.png">

<img width="1440" alt="Screenshot 2023-04-28 at 3 56 35 PM" src="https://user-images.githubusercontent.com/8112138/235071201-af26a6e4-771f-4b72-bc43-ac78f9bd65fa.png">

```suggestion
```

Reply:
Updated in https://github.com/MetaMask/metamask-extension/pull/18752/commits/bbe260718b04bf05c0074823520d157bb12a4938